mkt/extensions/views.py

```diff
@@ -54,10 +54,6 @@ def create(self, request, *args, **kwargs):
         if not upload_pk:
             raise exceptions.ParseError(_('No validation_id specified.'))
 
-        if not request.user.is_authenticated():
```

Comment:
This wasn't needed, all the views implementing this mixin use permissions. There are tests covering this.